src/status_im/ui/screens/screens.cljs

```diff
@@ -74,7 +74,12 @@
     [status-im.ui.screens.wallet.settings.views :as wallet-settings]
     [status-im.ui.screens.wallet.swap.views :as wallet.swap]
     [status-im.ui.screens.wallet.transactions.views :as wallet-transactions]
-    [status-im2.contexts.chat.group-details.view :as group-details]))
+    [status-im2.contexts.chat.group-details.view :as group-details]
+    [status-im2.contexts.chat.home.view :as chat-home]
```

Comment:
it's unclear why you have added all of these screens here? we already have them registered in the app? any particular reason? 🤨

Reply:
Want to navigate to the chat home screen. Just removed.